src/cluster/replication.cc

```diff
@@ -770,14 +770,16 @@ Status ReplicationThread::parallelFetchFile(const std::string &dir,
           if (this->stop_flag_) {
             return {Status::NotOK, "replication thread was stopped"};
           }
+          int sock_timeout_ms = 3000;
```

Comment:
I think it's better to move it to the top of the file and make it const/constexpr.

Reply:
ok, updated